kvcache/causal_test.go

```diff
@@ -305,6 +303,10 @@ func (b *testBackend) NewContext() ml.Context {
 	return &testContext{}
 }
 
+func (b *testBackend) NewContextSize(int) ml.Context {
+	return &testContext{}
+}
```

Comment:
I don't think this is used anywhere - maybe it is from another PR?

Reply:
I missed this when splitting from #9378